controller/controller.go

```diff
@@ -857,13 +862,14 @@ func (ctrl *ProvisionController) Run(ctx context.Context) {
 			go wait.Until(func() { ctrl.runVolumeWorker(ctx) }, time.Second, ctx.Done())
 		}
 
-		klog.Infof("Started provisioner controller %s!", ctrl.component)
+		logger.Info("Started provisioner controller!", "component", ctrl.component)
```

Comment:
The convention is to not end the message string with punctuation because the key/value pairs follow.